src/commands/service/newstart.ts

```diff
@@ -0,0 +1,39 @@
+import {flags} from '@oclif/command'
+import Command from '../../service-command'
+
+export default class ServiceDeployNew extends Command {
+  static description = 'Start a service'
+
+  static flags = {
+    ...Command.flags,
+    env: flags.string({
+      description: 'set env defined in mesg.yml (configuration.env)',
+      multiple: true,
+      helpValue: 'FOO=BAR'
+    })
+  }
+
+  static args = [{
+    name: 'SERVICE',
+    required: true,
+    description: 'Service\'s sid or hash'
+  }]
+
+  static hidden = true
+
+  async run(): Promise<string> {
+    const {args, flags} = this.parse(ServiceDeployNew)
+    this.spinner.start('Start service')
+    return new Promise<string>((resolve, reject) => {
+      this.instanceAPI.Create({id: args.SERVICE, env: flags.env}, (err, resp) => {
```

Comment:
let's try to use [promisify](https://nodejs.org/api/util.html#util_util_promisify_original) if we can it will be way nicer with `await promisfy(this.instanceAPI.create)({ ... })`. This can even be done when we create the instanceAPI.
Definitely not blocking but to think about next time ;)

Reply:
I'l consider this later and update whole cli with promisify